### PR TITLE
Es6 projects common modules video more in series container

### PR DESCRIPTION
**Pull request by svillafe**
## What does this change?

Converts to es6-style the module `more-in-series-container.js`

## Does this affect other platforms - Amp, Apps, etc?
I think just desktop.

## Tested in CODE?
Only in DEV

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/video/more-in-series-container.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        : ''}.json?shortUrl=${shortUrl
+    // exclude professional network content from video pages
+    }${mediaType === 'video'
+        ? '&exclude-tag=guardian-professional/guardian-professional'
+        : ''}`;
+
+    component.endpoint = endpoint;
+
+    component.fetch(el).then(() => {
+        checkElemsForVideos([el]);
```
Comment (SiAdcock) [inline static/src/javascripts/projects/common/modules/video/more-in-series-container.js]: Confirmed with @GHaberis, this is the correct fix for this issue

**Review by SiAdcock — APPROVED** [review]
🎉 Nice work, man

File: tools/es5to6.json [hunk 2]
```diff
@@ -170,8 +170,6 @@
     "projects/commercial/modules/paidfor-band.js"
   ],
   "Philip Wills": [],
-  "Santiago Villa Fernandez": [
-    "projects/common/modules/video/more-in-series-container.js"
-  ],
+  "Santiago Villa Fernandez": [],
```
Comment (gustavpursche) [inline tools/es5to6.json]: ✨

**Comment by paperboyo** [comment]
In the future this may (maybe!) be a good candidate tu use the new [CAPI feature](https://github.com/guardian/content-api/pull/1949). WDYT, @akash1810?

File: static/src/javascripts/projects/common/modules/video/more-in-series-container.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+const moreInSeriesContainerInit = (
+    el: HTMLElement,
+    mediaType: string,
+    section: string,
+    shortUrl: string,
+    series: string
+): void => {
+    const component = new Component();
+    const endpoint = `/${mediaType}/section/${section}${series
```
Comment (gustavpursche) [inline static/src/javascripts/projects/common/modules/video/more-in-series-container.js]: According to the flow definition `series` should always be truthy so I think this check could be removed or the type definition should be adjusted. What do you think?

Comment (svillafe) [inline static/src/javascripts/projects/common/modules/video/more-in-series-container.js]: Good catch! I will turn it into a [maybe string](https://flow.org/en/docs/types/maybe/).

**Comment by svillafe** [comment]
In the DEV is working.
picture 294

**Pull request merged**